src/AbstractActionHandler.ts

```diff
@@ -33,8 +35,10 @@ export abstract class AbstractActionHandler {
       console.info(`Rolling back ${rollbackCount} blocks to block ${rollbackBlockNumber}...`)
       await this.rollbackTo(rollbackBlockNumber)
       await this.refreshIndexState()
-    } else if (!this.lastProcessedBlockHash && this.lastProcessedBlockNumber === 0) {
+      await this.refreshHandlerVersionState()
+    } else if (this.lastProcessedBlockNumber === 0 && this.lastProcessedBlockHash === "") {
```

Comment:
I'm curious, why the change here?

Reply:
More explicit- the initial state of `this.lastProcessedBlockHash` is an empty string, so when reading the code it's easy to see that that's what I'm checking for.